src/consts.js

```diff
@@ -8,3 +8,6 @@ export const META_DESCRIPTION =
 export const META_KEYWORDS =
   'data version control machine learning models management'
 export const META_SOCIAL_IMAGE = 'https://dvc.org/static/social-share.png'
+
+export const FORUM_URL = 'https://discuss.dvc.org'
+export const BLOG_URL = 'https://blog.dvc.org'
```

Comment:
How about something like

```js
import hostname from 'os'

export const FORUM_URL = 'https://discuss.' + hostname()
export const BLOG_URL = 'https://blog.' + hostname()
```

Reply:
We can do it, but then it will not work on Netlify and on localhost. So I think we should hardcode dvc.org here.

Reply:
OK how about

```js
const WEBSITE_HOST = 'dvc.org'
export const FORUM_URL = 'https://discuss.' + WEBSITE_HOST
export const BLOG_URL = 'https://blog.' + WEBSITE_HOST
```

in case `WEBSITE_HOST` so its hardcoded in a single place?

Also, could `META_SOCIAL_IMAGE` below be just

```js
export const META_SOCIAL_IMAGE = '/static/social-share.png'
```

?

Reply:
We can add `WEBSITE_HOST`, yes, but `META_SOCIAL_IMAGE` has to be full url – it is not used on our site, it is used by facebook/twitter to display an image preview.

Reply:
since @jorgeorpinel is traveling today ... taking aa liberty to resolve and merge to move this thingy forward

Reply:
@jorgeorpinel feel free to reopen review and we will get to it again

Reply:
Looks perfect. Thanks!